### PR TITLE
Fixing IE scrolling issue and also fixing horizontal scrollbar

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys fixed the issue, have a look.

### CODE REVIEW

File: web/react/components/posts_view.jsx [hunk 1]
```diff
@@ -233,7 +233,8 @@ export default class PostsView extends React.Component {
             window.requestAnimationFrame(() => {
                 // If separator exists scroll to it. Otherwise scroll to bottom.
                 if (this.refs.newMessageSeparator) {
-                    this.refs.newMessageSeparator.scrollIntoView();
+                    var objDiv = this.refs.postlist;
+                    objDiv.scrollTop = this.refs.newMessageSeparator.offsetTop; //scrolls node to top of Div
```
Comment (jwilander) [inline web/react/components/posts_view.jsx]: What browsers has this been tested with? Will it work with all of them?

Comment (asaadmahmoodspin) [inline web/react/components/posts_view.jsx]: Tested it on IE, Chrome and Firefox.

Comment (asaadmahmoodspin) [inline web/react/components/posts_view.jsx]: Also just tested it on safari. Works fine.

**Comment by jwilander** [comment]
+1

**Comment by hmhealey** [comment]
+1

**Comment by asaadmahmoodspin** [comment]
Hold the merge, trying to fix the page up issue.

**Comment by asaadmahmoodspin** [comment]
Okay guys fixed the other issue too, we can merge now.

**Comment by coreyhulen** [comment]
+1

**Pull request merged**